tests/core/test_reference.py

```diff
@@ -184,25 +184,25 @@ class MySchema(BaseModel):
     assert new_ref.output_schema is None
 
     with pytest.raises(AttributeError):
-        assert ref.a.uuid == "123"
+        ref.a.uuid
 
     with pytest.raises(AttributeError):
-        assert ref["a"].uuid == "123"
+        ref["a"].uuid
 
     with pytest.raises(AttributeError):
-        assert ref[1].uuid == "123"
+        ref[1].uuid
 
     # check valid nested schemas
     assert ref.nested.s.uuid == "123"
     with pytest.raises(AttributeError):
-        assert ref.nested.m.uuid == "123"
+        ref.nested.m.uuid
     assert ref.nested.nested.n.uuid == "123"
     with pytest.raises(AttributeError):
-        assert ref.nested.nested.m.uuid == "123"
+        ref.nested.nested.m.uuid
 
     assert ref.nested.nested_opt.n.uuid == "123"
     with pytest.raises(AttributeError):
-        assert ref.nested.nested_opt.m.uuid == "123"
```

Comment:
@utf: Do you have any idea why `ref.nested.nested_opt.m.uuid` is getting defined as being `"123"` despite `.m` not being an attribute in the model?